xml/tuning_cgroups.xml

```diff
@@ -58,58 +58,33 @@
   <itemizedlist>
    <listitem>
     <para>
-     hybrid: v2 hierarchy without controllers, and the controllers are on v1
-     hierarchies
+     unified: v2 hierarchy with controllers
     </para>
    </listitem>
    <listitem>
     <para>
-     unified: v2 hierarchy with controllers
+     hybrid: v2 hierarchy without controllers, and the controllers are on v1
+     hierarchies (deprecated)
     </para>
    </listitem>
   </itemizedlist>
   <para>
-   The current default mode is hybrid. This
-   provides backwards compatibility for applications that need it.
-   The following features are available only with the unified v2 hierarchy:
+   The default mode is unified.
+   There is a hybrid mode that provides backwards compatibility for applications
```

Comment:
I leave it to your judgement whether to highlight the information about backward compatibility in some way. If you would like to highlight it, the easiest way would be to wrap it as `<emphasis>backward compatibility</emphasis>`.

Reply:
In my perspective the part to highlight is "applications that need it". So this can be kept as is, one unhighlighted sentence.